File: 1383/1383.cs
public class Solution {
    public int MaximumPerformanceOfATeam(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MaximumPerformanceOfATeam(root.left), MaximumPerformanceOfATeam(root.right));
    }
}

File: 1383/1383.py
class Solution:
    def maximum_performance_of_a_team(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.maximum_performance_of_a_team(root.left), self.maximum_performance_of_a_team(root.right))